src/NuGet.Clients/NuGet.VisualStudio.Common/Services/NuGetFeatureFlagService.cs

```diff
@@ -6,6 +6,7 @@
 using System.ComponentModel.Composition;
 using System.Threading.Tasks;
 using Microsoft.Internal.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Services.Common;
```

Comment:
Do we need this?

Reply:
that's where TryAdd comes from apparently 🤔

Reply:
Oh yeah, TryAdd isn't available on regular dict. 
Given that you're making the call on the UI thread, you can probably just call `[]`. 

I don't care too much, but it seems weird to bring in namespaces for something basic like that.